Dockerfile.bolt-server

```diff
@@ -10,7 +10,8 @@ RUN mkdir /bolt-server
 # Gemfile requires gemspec which requires bolt/version which requires bolt
 ADD . /bolt-server
 WORKDIR /bolt-server
-RUN bundle install --no-cache --path vendor/bundle
+RUN bundle config set path 'vendor/bundle' \
```

Comment:
I think this needs `&&`? `--no-cache` is _also_ deprecated, so seems like we should set that here as well.

Reply:
Added the `&&`. I'm not sure about no-cache. That option is a bit unusual because it doesn't accept any values. Also I didn't see any deprecation message during bundler.